src/internal/check-env.js

```diff
@@ -1,11 +1,16 @@
-const required = [ // this is the absolute minimum required to run wildbeast
-  'BOT_TOKEN',
-  'BOT_PREFIX',
-  'WILDBEAST_MASTERS'
-]
+const joi = require('@hapi/joi')
 
-for (let x of required) {
-  if (!process.env[x]) {
-    global.logger.error(`Missing environment variable ${x}`, true)
-  }
-}
+const schema = joi.object().keys({
+  BOT_TOKEN: joi.string().required(),
+  BOT_PREFIX: joi.string().required(),
+  SENTRY_DSN: joi.string(),
+  ENABLE_METRICS: joi.boolean(),
+  METRICS_INTERVAL: joi.number(),
+  WILDBEAST_MASTERS: joi.string().required().regex(/([0-9]{17,},?)\1?/)
+})
+
+const res = schema.validate(process.env, {
+  allowUnknown: true
+})
+if (process.env.WILDBEAST_ENV_CHECK_DISABLED && res.error) logger.warn('ENV', 'Found environment warnings, but checking is disabled!')
```

Comment:
Separate blocks with separate responsibilities...
```suggestion
const res = schema.validate(process.env, {
  allowUnknown: true
})

if (process.env.WILDBEAST_ENV_CHECK_DISABLED && res.error) logger.warn('ENV', 'Found environment warnings, but checking is disabled!')
```
or, in this case, maybe use a one-liner (personal preference):
```suggestion
const res = schema.validate(process.env, { allowUnknown: true })
if (process.env.WILDBEAST_ENV_CHECK_DISABLED && res.error) logger.warn('ENV', 'Found environment warnings, but checking is disabled!')
```